src/scripts/services/dimManifestService.factory.js

```diff
@@ -1,250 +1,250 @@
+      // indexedDB. We can remove this after a few releases, but we
+      // want to save disk for our users.
+      deleteOldManifestFile();
+
+      manifestPromise = dimBungieService.getManifest()
+        .then(function(data) {
+          const language = dimSettingsService.language;
+          const path = data.mobileWorldContentPaths[language] || data.mobileWorldContentPaths.en;
+
+          // Use the path as the version, rather than the "version" field, because
+          // Bungie can update the manifest file without changing that version.
+          const version = path;
+          service.version = version;
+
+          return loadManifestFromCache(version)
+            .catch(function(e) {
+              return loadManifestRemote(version, language, path);
+            })
+            .then(function(typedArray) {
+              service.statusText = $translate.instant('Manifest.Build') + '...';
+
+              return Promise.all([
+                typedArray,
+                requireSqlLib(), // load in the sql.js library
+              ]);
+            })
+            .then(function([typedArray, SQLLib]) {
+              const db = new SQLLib.Database(typedArray);
+              // do a small request, just to test it out
+              service.getAllRecords(db, 'DestinyRaceDefinition');
+              return db;
+            });
+        })
+        .catch((e) => {
+          let message = e.message || e;
+          if (e.status && e.status !== 200) {
+            message = $translate.instant('Manifest.BungieDown', { error: e.statusText });
+          }
+          service.statusText = $translate.instant('Manifest.Error', { error: message });
+          manifestPromise = null;
+          service.isError = true;
+          deleteManifestFile();
+          throw e;
+        });
+
+      return manifestPromise;
+    },
+
+    getRecord: function(db, table, id) {
+      const statement = makeStatement(table, db);
+      // The ID in sqlite is a signed 32-bit int, while the id we
+      // use is unsigned, so we must convert
+      const sqlId = new Int32Array([id])[0];
+      const result = statement.get([sqlId]);
+      statement.reset();
+      if (result.length) {
+        return JSON.parse(result[0]);
+      }
+      return null;
+    },
+
+    getAllRecords: function(db, table) {
+      const rows = db.exec(`SELECT json FROM ${table}`);
+      const result = {};
+      rows[0].values.forEach((row) => {
+        const obj = JSON.parse(row);
+        result[obj.hash] = obj;
+      });
+      return result;
+    }
+  };
+
+  return service;
+
+  /**
+   * Returns a promise for the manifest data as a Uint8Array. Will cache it on succcess.
+   */
+  function loadManifestRemote(version, language, path) {
+    service.statusText = $translate.instant('Manifest.Download') + '...';
+
+    return $http.get("https://www.bungie.net" + path + '?host=' + window.location.hostname, { responseType: "blob" })

```

Comment:
Fixed an issue with chrome caching origin. You bump into it when you visit a website during testing, and then view it on beta site. Host name forces recache based on subdomain.